### PR TITLE
warn the script is depreciated

### PR DESCRIPTION
**Pull request by jessfraz**
this should be in 1.8.0, since we will wait to merge https://github.com/docker/docker/pull/14745 til after

ping @tianon

### CODE REVIEW

File: hack/release.sh [hunk 1]
```diff
@@ -293,30 +293,8 @@ EOF
 	# Upload repo
 	s3cmd --acl-public "$s3Headers" sync "$APTDIR/" "s3://$BUCKET/ubuntu/"
 	cat <<EOF | write_to_s3 s3://$BUCKET/ubuntu/index
-echo "# WARNING! This script is not going to install aufs on your"
-echo "# system. If you want to use aufs please check the script"
+echo "# WARNING! This script is depreciated. Please use the script"
```
Comment (tianon) [inline hack/release.sh]: I think you meant `deprecated` here. :+1:

Comment (jfrazelle) [inline hack/release.sh]: spelling is hard hahaha

**Comment by calavera** [comment]
LGTM

File: hack/release.sh [hunk 2]
```diff
@@ -293,30 +293,8 @@ EOF
 	# Upload repo
 	s3cmd --acl-public "$s3Headers" sync "$APTDIR/" "s3://$BUCKET/ubuntu/"
 	cat <<EOF | write_to_s3 s3://$BUCKET/ubuntu/index
-echo "# WARNING! This script is not going to install aufs on your"
-echo "# system. If you want to use aufs please check the script"
+echo "# WARNING! This script is deprecated. Please use the script"
 echo "# at https://get.docker.com"
```
Comment (tiborvass) [inline hack/release.sh]: `https:/get.docker.com/` with a leading slash :)

**Comment by tianon** [comment]
LGTM

File: hack/release.sh (hunk 2 above)
Comment (jfrazelle) [inline hack/release.sh]: good catch! fixed

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**